packages/zambdas/src/ehr/get-chart-data/index.ts

```diff
@@ -78,10 +79,17 @@ export async function getChartData(
     defaultSearchBy?: 'encounter' | 'patient';
   }): void {
     const fieldOptions = requestedFields?.[field];
+    const defaultSearchParams = !requestedFields ? defaultChartDataFieldsSearchParams[field] : undefined;
 
     if (!requestedFields || fieldOptions) {
       chartDataRequests.push(
-        createFindResourceRequest(patient, encounter, resourceType, fieldOptions, defaultSearchBy)
+        createFindResourceRequest(
+          patient,
+          encounter,
+          resourceType,
+          fieldOptions || defaultSearchParams,
```

Comment:
Perhaps it would be better to keep this logic untouched and add only _tag to the default parameters, because this change will affect the subsequent logic.